dataclasses_jsonschema/field_types.py

```diff
@@ -27,6 +27,20 @@ def json_schema(self) -> JsonDict:
         raise NotImplementedError()
 
 
+class DateFieldEncoder(FieldEncoder[date, str]):
+    """Encodes dates to RFC3339 format"""
+
+    def to_wire(self, value: date) -> str:
+        return value.isoformat()
+
+    def to_python(self, value: str) -> date:
+        return value if isinstance(value, date) else date.fromisoformat(cast(str, value))
```

Comment:
Unfortunately, [`date.fromisoformat`](https://docs.python.org/3.7/library/datetime.html#datetime.date.fromisoformat) was only added in python 3.7 and I'd like to continue supporting python 3.6 for now.

Reply:
You should be able to use [dateutil](https://dateutil.readthedocs.io/en/stable/) instead which is already a dependency.

Reply:
Good point - sorry. Now updated, thanks.